pkg/workflow/client.go

```diff
@@ -406,8 +406,8 @@ type ArgoAPIClient struct {
 }
 
 func NewArgoAPIClient(restCfg *rest.Config) *ArgoAPIClient {
-	apiConfig := k8s.ToClientCmdApiConfig(restCfg)
-	clientConfig := clientcmd.NewDefaultClientConfig(apiConfig, nil)
+	apiConfig, _ := k8s.ToClientCmdApiConfig(restCfg)
```

Comment:
Why do we ignore the error?

Reply:
The reason is we follow the pattern of init API handler not return error. Let expose it when init API server.